statviz/src/App.tsx

```diff
@@ -141,6 +147,20 @@ function App() {
         <Heading size="md">Organization: {data.resolveLink.organisationName}</Heading>
         <Heading size="md">Base: {data.resolveLink.baseName}</Heading>
       </Flex>
+      <Flex
+        borderWidth="1"
+        padding="15"
+        marginBottom="15"
+        shadow="md"
+        justifyContent="left"
+        background="white"
+      >
+        <WrapItem w="150">
+          <Center>
+            <BoxesOrItemsSelect />
```

Comment:
please make it such that the used SelectField shows the label "show as:" instead of "display by:", and that the label and the control are on one line. But at the same time the SelectField should behave the same in all other places it's being used.